### PR TITLE
Add Generic pairing instructions

### PR DESCRIPTION
**Pull request by Spidy123222**
This adds generic pairing instructions for users and this would also allow us in sidestore app to replace that video in pairing step to not rickroll users that was previously added from one of our former contributors as an inside joke until instructions were made.

This includes also instructions for users on windows if the Jitterbugpair program doesn’t execute properly.

### CODE REVIEW

File: guides/install.md [hunk 1]
```diff
@@ -26,4 +26,22 @@ With SideStore downloader installed (and it's requirements met), simply connect
 You must then enable "Developer Mode" in your iOS security settings. Additionally, you must *Trust* your personal developer certificate.
 
 Finally, open the SideStore app on your homescreen, re-enter the Apple ID credentials used previously, and refresh to ensure that everything is working correctly.
--->
\ No newline at end of file
+-->
+
+# Pairing Process
+
+Using Jitterbugpair (not jitterbug app) allows us to create a pairing file for programs, like sidestore to talk to your device and is required to use sidestore or it will not function. (Windows and MacOS requires iTunes)
```
Comment (naturecodevoid) [inline guides/install.md:33]: Doesn't mac come with iTunes/Music reinstalled? And I think jitterbugpair just uses libimobiledevice. So is it even required on windows?

File: guides/install.md [hunk 2]
```diff
@@ -26,4 +26,22 @@ With SideStore downloader installed (and it's requirements met), simply connect
...
+
+[Windows (x64)](https://github.com/osy/Jitterbug/releases/download/v1.3.1/jitterbugpair-win64.zip)
+
+[MacOS](https://github.com/osy/Jitterbug/releases/download/v1.3.1/jitterbugpair-macos.zip)
+
+[Linux](https://github.com/osy/Jitterbug/releases/download/v1.3.1/jitterbugpair-linux.zip)
+
+Generic Pair Instructions:
+
+Unzip the Jitterbugpair program zip file. Plug your idevice into the computer and the idevice needs to be on the home screen and execute the program. Your idevice might get asked to trust this computer just trust it as it can be part of pairing process. After pairing you get a file that ends with ```.mobiledevicepairing``` which you can then transfer to your idevice via iTunes or other forms of transportation. It's also possible to change the extension to ```.plist``` for support with older sidestore versions.
```
Comment (naturecodevoid) [inline guides/install.md:45]: It might be good to turn this into a numbered list

**Review by naturecodevoid — APPROVED** [review]
Looks good apart from some very minor things. I don't think you need to use \`\`\`text\`\`\` for code blocks, you should be able to just do \`text\`. It might be good to add some sections for each platform:
- Mac: instructions on SideServer mac, info about auto pairing
- Windows: instructions for using AltServer to sideload the IPA, and a link to the pairing process (it should be `#pairing-process`)
- Linux: link to AltServer-Linux with basic instructions on how sideloading might be achieved, along with disclaimer that some people have had issues and a link to the pairing process section

**Comment by Spidy123222** [comment]
> Looks good apart from some very minor things. I don't think you need to use ```text``` for code blocks, you should be able to just do `text`. It might be good to add some sections for each platform:
> 
> * Mac: instructions on SideServer mac, info about auto pairing
> * Windows: instructions for using AltServer to sideload the IPA, and a link to the pairing process (it should be `#pairing-process`)
> * Linux: link to AltServer-Linux with basic instructions on how sideloading might be achieved, along with disclaimer that some people have had issues and a link to the pairing process section

Doing single back quotes didn’t seem to format properly and prob should mention sideserver on macOS but all the others are more relevant in a installation guide above what I’m writing which is purely for pairing not installing but it part of installation process if user didn’t use altserver based install. Linux altserver doesn’t seem to work from what I’m told for sideloading sidestore as it gets same effect as sideloady which would make it not work properly. I’m not sure what your referring with `#pairing-process`.

File: guides/install.md (hunk 1 above)
Comment (Spidy123222) [reply guides/install.md:33]: > Doesn't mac come with iTunes/Music preinstalled? And I think jitterbugpair just uses libimobiledevice. So is iTunes even required on windows?

Yes it’s required on windows. I’m just listing what jitterbugpair requires on their GitHub. I don’t have a mac to know that part and if older mac versions don’t need it.

**Comment by naturecodevoid** [comment]
> prob should mention sideserver on macOS but all the others are more relevant in a installation guide above what I’m writing which is purely for pairing not installing but it part of installation process if user didn’t use altserver based install.

You're editing the installation guide, why not add it all in one PR? You can rename the branch and PR name if that's why you want to separate it.

> Linux altserver doesn’t seem to work from what I’m told for sideloading sidestore as it gets same effect as sideloady which would make it not work properly.

When you write the installation guide, you should probably include this as a huge disclaimer.

> I’m not sure what your referring with `#pairing-process`.

In markdown, you can link to other sections with #<section id (not sure what to call it>. If you view the file on github and hover over a header and click the link button on the left, then look at the URL, which should end with the header/section "id".

File: guides/install.md (hunk 1 above)
Comment (Spidy123222) [reply guides/install.md]: Does latest changes fixes your problem?

**Comment by Spidy123222** [comment]
> > prob should mention sideserver on macOS but all the others are more relevant in a installation guide above what I’m writing which is purely for pairing not installing but it part of installation process if user didn’t use altserver based install.
> 
> You're editing the installation guide, why not add it all in one PR? You can rename the branch and PR name if that's why you want to separate it.
> 
> > Linux altserver doesn’t seem to work from what I’m told for sideloading sidestore as it gets same effect as sideloady which would make it not work properly.
> 
> When you write the installation guide, you should probably include this as a huge disclaimer.
> 
> > I’m not sure what your referring with `#pairing-process`.
> 
> In markdown, you can link to other sections with #<section id (not sure what to call it>. If you view the file on github and hover over a header and click the link button on the left, then look at the URL, which should end with the header/section "id".

This was discussed in the discord server but essentially will have a separate pull request that will do installation instructions. Getting a pairing file is part of instructions but shouldn’t be required when SideServer comes for some platforms.

**Review by naturecodevoid — APPROVED** [review]

**Pull request merged**